README.md

```diff
@@ -0,0 +1,35 @@
+# SONiC-restapi
+
+## Description
+This is a configuration agent which exposes HTTPS endpoints to perform dynamic network configuration on switches running SONiC. It restful API server is `go-server-server`
+
+## Getting Started
+### Fetch Dependencies
+  1. `mkdir debs; cd debs`
+  2. `sudo wget https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-all/lastSuccessfulBuild/artifact/target/debs/*zip*/debs.zip`
+  3. `sudo unzip debs.zip`
+  
+### Build Rest-API
+  1. Modify `copy.sh` SDIR path to point to your downloaded debs from the previous step
+  2. Execute ./build.sh
+  3. The above should generate 2 Docker images, 1 which is used for local development on your VM and 1 which is used for deployment on a TOR
+  4. Run `docker images` to check if rest-api dockers were generated <br/>
+      		<pre>`REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE` <br/>
```

Comment:
use ``` ``` for multiline code block.

https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html